EIPS/eip-2569.md

```diff
@@ -0,0 +1,357 @@
+---
+eip: 2569
+title: Saving and Displaying Image Onchain for Universal Tokens
+author: Hua Zhang (@dgczhh), Yuefei Tan (@whtyfhas), Derek Zhou (@zhous)
+discussions-to: https://ethereum-magicians.org/t/erc-2569-saving-and-displaying-image-onchain-for-universal-tokens/4167
+status: Draft
+type: Standards Track
+category: ERC
+created: 2020-03-28
+---
+
+## Simple Summary
+A set of interfaces to save an SVG image in Ethereum, and to retrieve the image file from Ethereum for universal tokens. 
+
+## Abstract
+This set of interfaces allow a smart contract to save an SVG image in Ethereum and to retrieve an SVG image from Ethereum for fungible tokens, non-fungible tokens and tokens based on standards that will be developed in the future. 
+
+The interface set has two interfaces: one to save an SVG file in Ethereum and the other to retrieve an SVG file from Ethereum. 
+
+Typical applications include but not limited to:
+A solution for storage of a fungible token's icon.
+A solution for storage of a non-fungible token's icon.
+A solution for storage of the icon/logo of a DAO's reputation token.
```

Comment:
These will render as one paragraph.  If you want a bulleted list you need:
```suggestion
* A solution for storage of a fungible token's icon.
* A solution for storage of a non-fungible token's icon.
* A solution for storage of the icon/logo of a DAO's reputation token.
```

Reply:
Changes made and committed